### PR TITLE
added basic deployment code, minor cleanups/adustments

### LINKED ISSUE #2: Work out basic smart contract deployment process/code

### PR DESCRIPTION
**Pull request by palmerss**

### CODE REVIEW

**Comment by palmerss** [comment]
this also adds basic skeleton structure for the timed smart contract item

**Comment by drewdrew12345** [comment]
Not really important but maybe we could use usual python filename conventions e.g. akita_inu_asa_utils.py instead of AkitaInuAsaUtils.py

**Comment by drewdrew12345** [comment]
> Not really important but maybe we could use usual python filename conventions e.g. akita_inu_asa_utils.py instead of AkitaInuAsaUtils.py

And same deal with various function names

File: contracts/TimedAssetLockContract/Deployment.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from algosdk.future import transaction
+from algosdk import account, mnemonic, logic
+
+import AkitaInuASAUtils as AIASAU
```
Comment (drewdrew12345) [inline contracts/TimedAssetLockContract/Deployment.py]: Maybe we could have

from AkitaInuASAUtils import *

And just remove the AIASAU from the calls

**Pull request merged**